.github/workflows/run-script.yml

```diff
@@ -9,20 +9,19 @@ jobs:
   run_script_job:
     runs-on: ubuntu-latest
     steps:
-      - name: Check out the code
-        uses: actions/checkout@v3
+      # TODO: consider replacing python/pip/update-web-metadata installs with docker image
```

Comment:
i can build a container. will that make the workflow faster?

Reply:
It's not going to get a ton faster, because it already doesn't take much time. It would save some bandwidth for pypi, but again not much. It's only worthwhile if it would also be useful for you in other ways.